src/_utils/mediaSizeProvider/index.tsx

```diff
@@ -10,14 +10,26 @@ export enum MediaSize {
 
 export type MediaSizeProviderProps = Readonly<{
   children: ReactNode
+  mediaSizeForTestsOnly?: MediaSize
```

Comment:
nit: An other solution (seen in open source projects) is to export a `TestMediaSizeProvider`/`MockMediaSizeProvider`/`MemoryMediaSizeProvider`. It avoid mistakes in production code.

Reply:
I will move forward with mediaSizeForTestsOnly to unblock my test PRs. But we can easily migrate to a fake implementation later (we will have only one usage in kairos for this test param).
As well, mistake wise, if somebody can use a mediaSizeForTestsOnly props in prod, they can use a fake implemetation too :) unless we combine it with some test-linting rules (which we don't have yet).

I'm wondering how much test-only code is shipped in production binary as we speak :)